src/organizercore.cpp

```diff
@@ -1543,23 +1551,18 @@ void OrganizerCore::directory_refreshed()
     log::debug("structure deleter thread done");
   });
 
-  m_DirectoryUpdate = false;
-
   log::debug("clearing caches");
   for (int i = 0; i < m_ModList.rowCount(); ++i) {
     ModInfo::Ptr modInfo = ModInfo::getByIndex(i);
     modInfo->clearCaches();
   }
 
-  if (!m_PostRefreshTasks.empty()) {
-    log::debug("running {} post refresh tasks", m_PostRefreshTasks.size());
```

Comment:
Why was the `m_DirectoryUpdate = False` moved below the clearing of the mod conflict caches?

Reply:
No particular reason except that I don't see why it would be before - If post-refresh tasks need to access conflict info from mod, I think it's better to wait for the cache to be cleared.

It's very unlikely to call `onNextRefresh` in between these lines, but that could happen from a separate thread - That actually could happens during the signal dispatch below, so maybe I will see if I can prevent this, although this is going to get complicated for probably not much.

Reply:
Ok, wanted to know for posterity whether it was to solve a specific issue or if just because it seemed to make more sense.
There is a chance it was put before for some reason, guess we will find out if that was the case.